controller/backup_target_controller.go

```diff
@@ -307,6 +323,32 @@ func (btc *BackupTargetController) reconcile(name string) (err error) {
 		}
 	}
 
+	// start a BackupStoreTimer and keep it in a map[string]*BackupStoreTimer
+	stopTimer := func() {
```

Comment:
Wouldn't it be better to make "name" an argument of this function?

Reply:
Yes, it would be more clear.
Modified. Thanks